experimental/dds/tree2/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -47,6 +47,7 @@ import {
 } from "../../util";
 import { cursorForJsonableTreeNode, jsonableTreeFromCursor } from "../treeTextCursor";
 import { MemoizedIdRangeAllocator } from "../memoizedIdRangeAllocator";
+import { RevisionTagCodec } from "../../shared-tree-core";
```

Comment:
Doesn't our linter complain about the ordering of imports here?

Reply:
Now that you point it out I'm also surprised it didn't complain about this. I wonder if `../` imports aren't enforced as strictly? I moved this up to where it "looks right."